kpatch-build/create-diff-object.c

```diff
@@ -4081,6 +4102,23 @@ static void kpatch_find_func_profiling_calls(struct kpatch_elf *kelf)
 				insn[4] == 0x00 && insn[5] == 0x00)
 				sym->has_func_profiling = 1;
 			break;
+		case LOONGARCH64:
+			struct section *sec;
```

Comment:
Older gcc complains about this:
```
gcc -std=gnu11 -MMD -MP -I../kmod/patch -Iinsn -Wall -Wsign-compare -Wconversion -Wno-sign-conversion -g -Werror   -c -o create-diff-object.o create-diff-object.c
create-diff-object.c: In function ‘kpatch_find_func_profiling_calls’:
create-diff-object.c:4106:4: error: a label can only be part of a statement and a declaration is not a statement
    struct section *sec;
    ^~~~~~
```
if this new variable remains in v3, please wrap the case block in {} brackets to appease those compilers.